packages/core/utils/lib/sanitize/visitors/index.js

```diff
@@ -1,6 +1,24 @@
 'use strict';
 
+const composeVisitors = (visitors) => {
+  /*
+    reduceRight is used to iterate through the array of visitors
+    in reverse order, starting with the last method in the array.
+    The initial value for the reduction is an empty function
+    that serves as the final "next" function in the sequence.
+  */
+  return visitors.reduceRight(
+    (next, visitor) => {
+      return (...args) => {
+        return visitor(...args, next);
+      };
+    },
+    () => {}
+  );
+};
```

Comment:
Another option could be doing: 
```js
  return visitors.reduceRight(
    (next, visitor) => {
      return (visitorOptions, visitorUtils) => {
        const { data, key } = visitorOptions;
        visitor(visitorOptions, visitorUtils);
        // If the visitor removed the key, don't call the next visitor
        if (key in data) next();
      };
    },
    () => {}
  );
```

So the visitors do not know about the next() function.
Will be waiting for your opinion @Convly @alexandrebodin  :)

Reply:
I like the second option better, forcing visitors to know about next would be annoying to implement.
I think this would be a good temporary solution